jaq-json/tests/funs.rs

```diff
@@ -77,3 +77,50 @@ fn tojson() {
     give(json!(0), "0.0 / 0.0 | tojson", json!("null"));
     give(json!(0), "1.0 / 0.0 | tojson", json!("null"));
 }
+
+#[test]
+fn math_rem() {
+    // generated with this command with modification for errors and float rounding
+    // cargo run -- -rn 'def f: -2, -1, 0, 2.1, 3, 4000000001; f as $a | f as $b | "give!(json!(null), \"\($a) / \($b)\", \(try ($a % $b) catch tojson));"'
+    // TODO: use fail!()?
```

Comment:
as it was hard to follow what test results in what value i switch to using give! instead

Reply:
Wow, thank you so much for your effort to create these tests!